src/test/java/com/williamfiset/algorithms/strings/BoyerMooreStringSearchTest.java

```diff
@@ -36,11 +36,14 @@ public void shouldReturnOneOccurrence() {
 
   @Test
   public void shouldReturnMultiplyOccurrences() {
+    assertThat(underTest.findOccurrences("SAAT TE", "TE")).containsExactly(5);
     assertThat(
             underTest.findOccurrences("Sample text for testing the Boyer-Moore algorithm.", "te"))
         .containsExactly(7, 16);
     assertThat(underTest.findOccurrences("Sample text for testing the Boyer-Moore algorithm.", " "))
         .containsExactly(6, 11, 15, 23, 27, 39);
+    assertThat(underTest.findOccurrences("AABAACAADAABAABA", "AABA")).containsExactly(0, 9, 12);
+    assertThat(underTest.findOccurrences("AAAAAAA", "AA")).containsExactly(0, 1, 2, 3, 4, 5);
   }
```

Comment:
To make sure the Boyer Moore impl is 100% correct, can also add a test that verifies a long string against a brute force implementation that finds patterns a text?

Reply:
Yeah sure, this makes sense. I will add a brute force implementation in the test and randomly generate a large string that will validate the testing in a dynamic way.

Reply:
Thanks!